pyigm/guis/igmguesses.py

```diff
@@ -1421,13 +1435,13 @@ def __init__(self, component=None, parent=None):
         self.parent = parent
         #if not status is None:
         #    self.statusBar = status
-        self.label = QtGui.QLabel('Component:',self)
+        self.label = QLabel('Component:',self)
         self.zwidget = ltgsm.EditBox(-1., 'zc=', '{:0.5f}')
```

Comment:
Yeah, this is the new functionality of EditBox.  If you can figure
out how to stop it, we should change it.